src/Command/RecipesCommand.php

```diff
@@ -0,0 +1,256 @@
+    {
+        $this->flex = $flex;
+        $this->symfonyLock = $symfonyLock;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('symfony:recipes')
+            ->setAliases(['recipes'])
+            ->setDescription('Shows information about all available recipes.')
+            ->setDefinition([
+                new InputArgument('package', InputArgument::OPTIONAL, 'Package to inspect, if not provided all packages are.'),
+            ])
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $installedRepo = $this->getComposer()->getRepositoryManager()->getLocalRepository();
+
+        // Inspect one or all packages
+        $package = $input->getArgument('package');
+        if (null !== $package) {
+            $packages = [0 => ['name' => strtolower($package)]];
+        } else {
+            $locker = $this->getComposer()->getLocker();
+            $lockData = $locker->getLockData();
+
+            // Merge all packages installed
+            $packages = array_merge($lockData['packages'], $lockData['packages-dev']);
+        }
+
+        $operations = [];
+        foreach ($packages as $value) {
+            if (null === $pkg = $installedRepo->findPackage($value['name'], '*')) {
+                $this->getIO()->writeError(sprintf('<error>Package %s is not installed</error>', $value['name']));
+
+                continue;
+            }
+
+            $operations[] = new InformationOperation($pkg);
+        }
+
+        $recipes = $this->flex->fetchRecipes($operations);
+        ksort($recipes);
+
+        $nbRecipe = \count($recipes);
+        if ($nbRecipe <= 0) {
+            $this->getIO()->writeError('<error>No recipe found</error>');
+
+            return 1;
+        }
+
+        // Display the information about a specific recipe
+        if (1 === $nbRecipe) {
+            $this->displayPackageInformation(current($recipes));
+
+            return 0;
+        }
+
+        // display a resume of all packages
+        $write = [
+            '',
+            '<bg=blue;fg=white>                      </>',
+            '<bg=blue;fg=white> Recipes available.   </>',
+            '<bg=blue;fg=white>                      </>',
+            '',
+        ];
+
+        /** @var Recipe $recipe */
+        foreach ($recipes as $name => $recipe) {
+            $lockRef = $this->symfonyLock->get($name)['recipe']['ref'] ?? null;
+
+            $additional = '';
+            if (null === $lockRef && null !== $recipe->getRef()) {
+                $additional = '<comment>(recipe not installed)</comment>';
+            } elseif ($recipe->getRef() !== $lockRef) {
+                $additional = '<comment>(update available)</comment>';
+            }
+            $write[] = sprintf(' * %s %s', $name, $additional);
+        }
+
+        $write[] = '';
+        $write[] = '<fg=blue>Run</>:';
```

Comment:
```suggestion
        $write[] = 'Run:';
```

The blue color doesn't really add much here - and it's such a short word (and now I'm highlighting the below in green so less color here looks better)